doc/changelog/02-changed/1250-archimedean-mixin.md

```diff
@@ -1,4 +1,5 @@
 - in `archimedean.v`
-  + definition of `ceil` (use lemma `ceilNfloor` explicitly)
+  + the definition of archimedean structures now include `Num.floor` and `Num.ceil`
+  + as its consequence, `Num.ceil x = - Num.floor (- x)` does not hold definitionally anymore (use lemma `ceilNfloor` instead)
     ([#1250](https://github.com/math-comp/math-comp/pull/1250),
     by Kazuhiko Sakaguchi).
```

Comment:
It's not at all specific to this PR, but I found it a bit weird that the PR number and the attribution belong to the last item when we have several items like this.

Reply:
Putting attributions in the changelog seems to cause some problems [1], maybe we can think of not putting them inside the changelog, after all this is not the main purpose of the changelog to record this information, yet the PR link and the git hashes are here to trace it back if necessary

[1] Speaking of attributions, there is also the issue of backports from MathComp-Analysis and other libraries, attributions are even harder to trace back in these situations

Reply:
@affeldt-aist Can you open an issue and (add it to the topics for the next meeting, if discussing it in two weeks is not too late)?